Sakura/core/ai.py

```
@@ -0,0 +1,264 @@
+
+def initialize_gemini_model():
+    """Initialize the Gemini model"""
+    global gemini_model
+    if not GEMINI_API_KEY:
+        logger.error("❌ GEMINI_API_KEY not found in environment variables")
+        return
+    try:
+        genai.configure(api_key=GEMINI_API_KEY)
+        gemini_model = genai.GenerativeModel('gemini-2.5-flash')
+        logger.info("✅ Gemini model initialized successfully")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)